### ISSUE TITLE
Explore Re-ID models for finetuning on AIC20

### ISSUE THREAD

**Issue by hector6298** [hector6298/mcmt_gnn#21]

**Comment by hector6298** [comment]
# Search and Pruning

Objective: Create a small training set from source pool that can train a model with high accuracy on target data.



![Image](https://github.com/hector6298/mcmt_gnn/assets/41920808/775fa55e-98ed-4411-9952-6228f3b8c76e)


Look at the size of the blob (it represent the number of identities), it is comparable to other datasets like Market and Duke, but achieves higher accuracy
